docs/eventing/getting-started.md

```diff
@@ -0,0 +1,528 @@
+```
+
+To show the various types of events you can send, you will make three requests.
+
+1. To make the first request, which creates an event that has the `type` `greeting`, run the following in the SSH terminal:
+
+    ```sh
+    curl -v "default-broker.event-example.svc.cluster.local" \
+      -X POST \
+      -H "Ce-Id: say-hello" \
+      -H "Ce-Specversion: 0.2" \
+      -H "Ce-Type: greeting" \
+      -H "Ce-Source: not-sendoff" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello Knative!"}'
+    ```
+
+    Result: When your event is sent to the `Broker`, `hello-display` will activate and send it to the event consumer of the same name.
+
+    If the event has been received, you will receive a `202 Accepted` response similar to the one below:
+    ```sh
+    < HTTP/1.1 202 Accepted
+    < Content-Length: 0
+    < Date: Mon, 12 Aug 2019 19:48:18 GMT
+    ```
+
+
+2. To make the second request, which creates an event that has the `source` `sendoff`, run the following in the SSH terminal:
+
+    ```sh
+    curl -v "default-broker.event-example.svc.cluster.local" \
+      -X POST \
+      -H "Ce-Id: say-goodbye" \
+      -H "Ce-Specversion: 0.2" \
+      -H "Ce-Type: not-greeting" \
+      -H "Ce-Source: sendoff" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Goodbye Knative!"}'
+    ```
+
+   Result: When your event is sent to the `Broker`, `goodbye-display` will activate and send the event to the event consumer of the same name.
+
+    If the event has been received, you will receive a `202 Accepted` response similar to the one below:
+    ```sh
+    < HTTP/1.1 202 Accepted
+    < Content-Length: 0
+    < Date: Mon, 12 Aug 2019 19:48:18 GMT
+    ```
+
+
+3. To make the third request, which creates an event that has the `type` `greeting` and the`source` `sendoff`, run the following in the SSH terminal:
+
+    ```sh
+    curl -v "default-broker.event-example.svc.cluster.local" \
+      -X POST \
+      -H "Ce-Id: say-hello-goodbye" \
+      -H "Ce-Specversion: 0.2" \
+      -H "Ce-Type: greeting" \
+      -H "Ce-Source: sendoff" \
+      -H "Content-Type: application/json" \
+      -d '{"msg":"Hello Knative! Goodbye Knative!"}'
+   ```
+
+   Result: When your event is sent to the `Broker`,  `hello-display` and `goodbye-display` will activate and send the event to the event consumer of the same name.
+
+   If the event has been received, you will receive a `202 Accepted` response similar to the one below:
+    ```sh
+    < HTTP/1.1 202 Accepted
+    < Content-Length: 0
+    < Date: Mon, 12 Aug 2019 19:48:18 GMT
+    ```
+
+4. Exit SSH by typing `exit` into the command prompt.
+    
+If everything has been done correctly, you should have sent 2 events to the `hello-display` event consumer and 2 events to the `goodbye-display` event consumer (note that `say-hello-goodbye` is sent to *both* `hello-display` and `goodbye-display`). You will verify that these events were received correctly in the next section.
```

Comment:
See above comment about "should."

I'd also cut out "If everything has been done correctly".